octoprint_mrbeam/static/js/material_settings.js

```diff
@@ -72,6 +73,8 @@ $(function () {
                     "Use opaque acrylic in red or black to create nice objects and signs. Acrylic is great for outdoor applications."
                 ),
                 hints: "",
+                type: "product",
+                url: gettext("https://www.mr-beam.org/products/mr-beam-acryl-schwarz")
```

Comment:
shouldn't the default be the english link?

Reply:
done